### DISCUSSION TITLE
Motion sensitivity [Resolved - human error]

### DISCUSSION THREAD

**Discussion by olinorwell** [roleoroleo/yi-hack-MStar#463] [Q&A] [answered]
Is there a way to edit the scripts to lower the motion detection sensitivity?

I'm getting never ending false positives due to flies. I want it to be so low that it would take a very large object (i.e. a human) to trip the sensor.

I've looked in the UI and the scripts on the camera - I can't see anything obvious - is it inside a binary that came with the camera then?

**Comment by roleoroleo** [comment, marked as answer]
Which script?
You can find the motion sensitivity in the web interface -> camera settings.
![immagine](https://user-images.githubusercontent.com/39277388/198066132-a5f35a77-a49d-4f10-b942-2f640143f008.png)

**Reply by olinorwell** [reply]
Yeah I had it on 'Low'. I wondered if there was an even lower setting.

Although - I think somehow the permanent recording option got set, so perhaps that's why it is recording every minute seemingly.

Sorry so yes I think it's most likely my mistake.

**Reply by olinorwell** [reply]
I think that was definitely the issue - sorry for the confusion! It is no longer recording constantly.

I was trying to get the 'load' value to be less so changing everything, it always seems to be 2.5-3 on my h201c (Dome) device. Maybe that's fine, I guess the CPU may have multiple cores.

**Reply by roleoroleo** [reply]
> I think that was definitely the issue - sorry for the confusion! It is no longer recording constantly.

No problem.

> I was trying to get the 'load' value to be less so changing everything, it always seems to be 2.5-3 on my h201c (Dome) device. Maybe that's fine, I guess the CPU may have multiple cores.

Also in my cam the value is around 3. It's "normal".
